### PR TITLE
Added chatbot feature with updated requirement.txt

### PR DESCRIPTION
**Pull request by ayushi25jaiswal**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -9,10 +9,12 @@
 from wtforms.validators import DataRequired, URL
 from flask_ckeditor import CKEditor, CKEditorField
 from datetime import date
-from camera import VideoCamera
+# from camera import VideoCamera
```
Comment (Isha307) [inline app.py]: Why did you commented this here?

Comment (ayushi25jaiswal) [reply app.py]: Extremely sorry for that , i have corrected it can you please review it now.

Comment (ayushi25jaiswal) [reply app.py]: Screenshot (41)
i am getting this error in camera.py module

Comment (ayushi25jaiswal) [reply app.py]: Extremely sorry for the inconvenience, i have submitted new PR ,can you please review it again.

**Pull request closed without merging**